doc/date.md

```diff
@@ -0,0 +1,61 @@
+# Date
+
+`Napi::Date` class is a representation of the JavaScript `Date` object. The
+`Napi::Date` class inherits its behavior from `Napi::Value` class
+(for more info see [`Napi::Value`](value.md))
+
+## Methods
+
+### Constructor
+
+Creates a new _empty_ instance of a `Napi::Date` object.
+
+```cpp
+Napi::Date();
+```
+
+Returns a new _empty_ JavaScript `Napi::Date`.
+
+### New
+
+Creates a new instance of a `Napi::Date` object.
+
+```cpp
+static Napi::Date Napi::Date::New(Napi::Env env, double value);
+```
+
+ - `[in] env`: The environment in which to construct the `Napi::Date` object.
+ - `[in] value`: The value the JavaScript `Date` will contain
+
+Returns a new JavaScript `Napi::Date`.
+
+### NumberValue
+
+```cpp
+double Napi::Date::NumberValue() const;
+```
+
+Returns the `double` primitive equivalent of the given JavaScript
```

Comment:
Maybe this definition is better:
Returns the time value as `double` primitive  represented as milliseconds since midnight at the beginning of 01 January, 1970 UTC.